src/utils/validators_enhanced.py

```
@@ -0,0 +1,479 @@
+    'admin', 'administrator', 'root', 'system', 'test', 'guest', 'user',
+    'mod', 'moderator', 'support', 'help', 'api', 'www', 'mail', 'email',
+    'bot', 'null', 'undefined', 'delete', 'create', 'update', 'select',
+    'insert', 'drop', 'table', 'database', 'server', 'gm', 'gamemaster',
+    'console', 'world', 'guild', 'player', 'character', 'account'
+}
+
+# Common weak passwords to reject
+WEAK_PASSWORDS: Set[str] = {
+    'password', '123456', '123456789', 'qwerty', 'abc123', 'password123',
+    'admin', 'letmein', 'welcome', 'monkey', '1234567890', 'dragon',
+    'superman', 'michael', 'password1', 'superman', 'qwerty123',
```

Ruff (0.14.10)

Sets should not contain duplicate item `'superman'`

Remove duplicate item

(B033)